src/components/IndexFilterWrapperAndList.js

```diff
@@ -2,10 +2,9 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { FlatList } from 'react-native';
 
-import { Touchable } from '../Touchable';
-import { WrapperHorizontal } from '../Wrapper';
-import { RegularText } from '../Text';
-
+import { Touchable } from './Touchable';
+import { WrapperHorizontal } from './Wrapper';
+import { RegularText } from './Text';
```

Comment:
`IndexFilterElement.js` was also missing adjusted imports as well as `BB-BUS/IndexFilter.js`

![Bildschirmfoto 2021-12-03 um 16 22 40](https://user-images.githubusercontent.com/1942953/144627435-69fcd8d0-3df9-49fc-880a-c74ef7001f85.png)

i amended the changes and force pushed

Reply:
good catch. i have no idea why it did not cause issues while i tested it on my simulator. i must have been in an unsaved state.